src/components/ReviewCard.jsx

```diff
@@ -0,0 +1,12 @@
+const ReviewCard = ({ title, category, img, body, votes }) => {
```

Comment:
IMO it's neater to send the whole object through on props and destructure it our here. Only if oyu want to though :)